### ISSUE TITLE
Hints when using executeSql are mishandled by splitSql

### ISSUE THREAD

**Issue by alondhe** [OHDSI/SqlRender#73]
It looks like hints (e.g. "--HINT DISTRIBUTE_ON_KEY(analysis_id)") aren't being handled correctly when using executeSql. The call to splitSql seems to alter the double hyphen comment into a single hyphen, which then throws a SQL error, **([Amazon](500310) Invalid operation: syntax error at or near "-"**;

For example,
```
--HINT DISTRIBUTE_ON_KEY(analysis_id)
CREATE TABLE results.achilles_results_dist
...
```
becomes 
```
-HINT DISTRIBUTE_ON_KEY(analysis_id)
CREATE TABLE results.achilles_results_dist
...
```

I've been seeing this recently, but it wasn't a problem maybe 2 months ago.

**Comment by schuemie** [comment]
I can reproduce the problem. I don't see how it would have worked in the past, but I think I have fixed it now: 752f138185b179545078d90caac85c86d234d21e

Could you give the new version a try?

**Comment by alondhe** [comment]
Looks good, thanks! Closing this.
